cmd/playground/window_state.go

```diff
@@ -0,0 +1,72 @@
+package playground
+
+import (
+	"time"
+
+	"github.com/odpf/optimus/job"
+)
+
+type state struct {
+	windowV1      job.WindowV1
+	windowV2      job.WindowV2
+	scheduledTime time.Time
+}
+
+func (s *state) incrementScheduleTimeOn(increaseBy cursorPointer) {
+	switch increaseBy {
+	case pointToMinute:
+		s.scheduledTime = s.scheduledTime.Add(time.Minute)
+	case pointToHour:
+		s.scheduledTime = s.scheduledTime.Add(time.Hour)
+	case PointToDay:
+		s.scheduledTime = s.scheduledTime.AddDate(0, 0, 1)
+	case pointToMonth:
+		s.scheduledTime = s.scheduledTime.AddDate(0, 1, 0)
+	case pointToYear:
+		s.scheduledTime = s.scheduledTime.AddDate(1, 0, 0)
+	}
+}
+
+func (s *state) decrementScheduleTimeOn(decreaseBy cursorPointer) {
```

Comment:
also this, change `decreaseBy` into `cursor`

Reply:
this is actually already mentioned in the previous comment. let me know if it's better to be implemented or there's better idea.